src/tyr/route_serializer_valhalla.cc

```diff
@@ -621,7 +638,7 @@ void legs(const valhalla::Api& api, int route_index, rapidjson::writer_wrapper_t
   writer.end_array(); // legs
 }
 
-std::string serialize(const Api& api) {
+std::string serialize(Api& api) {
```

Comment:
Needed in order to get an `EnhancedTripLeg` for getting the previous edge end heading, as in the osrm serializer.